raiden/network/transport/matrix/transport.py

```diff
@@ -321,6 +321,14 @@ def _http_retry_delay() -> Iterable[float]:
 
         self._message_handler: Optional[MessageHandler] = None
 
+    @classmethod
+    def unwrap(cls, transport_message: TransportMessage) -> (Message, QueueIdentifier):
```

Comment:
the first tought that came to my mind was this "unwrap what? a transport?" 

i think unrwap_message looks better

Reply:
isn't it implicit in the parameter name?

Reply:
I also think that since we have the wrap method on TransportMessage we should have this unwrap method there too

A TransportMessage knows how to unwrap itself, i dont think is a transport responsability

Reply:
you're right, fixed

Reply:
yeah, both unwrap and unwrap_message are ok to me, not sure about the location of this method

Reply:
if the message knows how to wrap and unwrap itself, then i think it should be a method of the message class

Reply:
agree